users/views.py

```diff
@@ -56,10 +59,39 @@ def UserCreateConfirmView(request, token):
         return redirect('users:signup')
```

Comment:
The user should be redirected to `users:login` instead of `users:signup` in line 57, i.e. when clicking on the mailed confirmation link.
(Can't comment on line 57 - outside the diff :P)